.circleci/config.yml

```diff
@@ -0,0 +1,46 @@
+version: 2.1
+orbs:
+  docker: circleci/docker@2.1.1
+  sonarcloud: sonarsource/sonarcloud@2.0.0
+jobs:
+  test:
```

Comment:
I don't know if we should call this step "test" it's not actually running any test, should it be called "build"?